examples/purchaseTesterTypescript/app/screens/HomeScreen.tsx

```diff
@@ -134,6 +134,15 @@ const HomeScreen: React.FC<Props> = ({
       console.log("error")
     }
   }
+
+  const showInAppMessages = async() => {
+    try {
+      await Purchases.showInAppMessages();
+      console.log("Shown messages successfully");
+    } catch {
+      console.log("error");
```

Comment:
I know this is just a tester app, but maybe:
```suggestion
      console.log("Error showing in-app messages: ${error}");
```